packages/manager/apps/telecom/src/app/telecom/telephony/service/fax/customDomains/custom-domains.controller.js

```diff
@@ -21,9 +21,7 @@ export default /* @ngInject */ function TelecomTelephonyServiceFaxCustomDomainsC
   =============================== */
 
   function fetchDomains() {
-    return OvhApiDomain.v7()
-      .query()
-      .execute().$promise;
+    return $http.get('/domain').then(({ data }) => data);
```

Comment:
No expand needed ?

Reply:
The response of the v7 api is a `String[]` so it seems we don't have to expand it